src/components/DropdownFilterModal/DropdownFilterModal.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useState, useEffect, FunctionComponent } from "react";
+import {
+  Modal,
+  StyleSheet,
+  Text,
+  View,
+  Dimensions,
+  TextInput,
+  FlatList,
+  TouchableOpacity
+} from "react-native";
+import { size, fontSize, borderRadius, color } from "../../common/styles";
+import { AppText } from "../Layout/AppText";
+import { TopBackground } from "../Layout/TopBackground";
+import { AppMode } from "../../context/config";
+
+const styles = StyleSheet.create({
+  modalView: {
+    paddingHorizontal: 0,
+    marginBottom: 0,
+    padding: 0,
+    width: Dimensions.get("window").width,
+    height: Dimensions.get("window").height,
+    backgroundColor: "white",
+    borderRadius: 20,
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 2
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 5
```

Comment:
Can we use the `shadow` styling helper here?

Reply:
Have removed as per comment above